specification/maps/data-plane/Weather/preview/1.0/weather.json

```diff
@@ -88,6 +88,149 @@
         ]
       },
       "x-ms-parameter-location": "method"
+    },
+    "WeatherYear": {
+      "name": "year",
+      "description": "Year of the cyclone(s)",
+      "type": "integer",
+      "format": "int32",
+      "required": true,
+      "in": "query",
+      "x-ms-parameter-location": "method"
+    },
+    "RequiredWeatherBasinID": {
+      "name": "basinID",
+      "type": "string",
+      "description": "Basin identifier",
+      "required": true,
+      "in": "query",
+      "enum": [
+        "AL",
+        "EP",
+        "SI",
+        "NI",
+        "CP",
+        "NP",
+        "SP"
+      ],
+      "x-ms-enum": {
+        "name": "basinID",
+        "modelAsString": true,
+        "values": [
+          {
+            "value": "AL",
+            "description": "AL"
```

Comment:
I think we can use more detailed description here 

![image](https://user-images.githubusercontent.com/6958125/130040808-6ff80956-4bd5-449e-8113-3d2e946dc626.png)

Reply:
Thanks for sharing the description. 
I have updated them to the file.